vendor/engines/sanger_sequencing/app/presenters/sanger_sequencing/well_plate_presenter.rb

```diff
@@ -2,20 +2,23 @@ module SangerSequencing
 
   class WellPlatePresenter
 
-    attr_reader :well_plate
+    include TextHelpers::Translation
+
+    attr_reader :well_plate, :plate_mode
     delegate :samples, to: :well_plate
 
-    def initialize(well_plate)
+    def initialize(well_plate, plate_mode = :default)
       @well_plate = well_plate
+      @plate_mode = plate_mode
     end
 
     def to_csv
       CSV.generate do |csv|
         csv << ["Container Name", "Plate ID", "Description", "ContainerType", "AppType", "Owner", "Operator", "PlateSealing", "SchedulingPref"]
-        csv << ["",               "",         "",            "96-Well",       "Regular", "mb core", "mbcore", "Septa", "1234"]
+        csv << ["",               "",         "",            "96-Well",       "Regular", text("owner"), text("operator"), "Septa", "1234"]
```

Comment:
Doesn't necessarily have to happen immediately but any benefit in I18n-ing the other fields?